tests/jvm/src/test/scala-2.13/scala/meta/tests/api/SurfaceSuite.scala

```diff
@@ -8,10 +8,13 @@ import scala.reflect.runtime.universe._
 import scala.reflect.runtime.{universe => ru}
 
 class SurfaceSuite extends FunSuite {
-  object CoreReflection extends {
-    val u: ru.type = ru
-    val mirror: u.Mirror = u.runtimeMirror(classOf[scala.meta.Tree].getClassLoader)
-  } with scala.meta.internal.trees.Reflection with scala.meta.internal.tokens.Reflection
+  object CoreReflection
+      extends {
```

Comment:
so you didn't use parent constructors = always? i thought that was the purpose of the change we merged

Reply:
I thought that this split before `extends` shouldn't depend on `binPack.parentConstructors` setting.
But then I've checked PR one more time and I found that it appears in case of `binPack.parentConstructors = Oneline` that we are currently using :disappointed: 

I decided just to go ahead as we already spent o lot of time on waiting fixes.